umh.docs.umh.app/content/en/docs/features/datainfrastructure/unified-namespace.md

```diff
@@ -1,67 +1,114 @@
 ---
```

Comment:
There are some ADRs here, which you could use to explain more why we chose something as the UNS:

https://learn.umh.app/lesson/data-modeling-in-the-unified-namespace-mqtt-kafka/